interpreter/llvm/src/tools/clang/lib/Sema/SemaOpenMP.cpp

```diff
@@ -105,7 +105,7 @@ class DSAStackTy {
           ConstructLoc(), OrderedRegion(false), InnerTeamsRegionLoc() {}
   };
 
-  typedef SmallVector<SharingMapTy, 64> StackTy;
+  typedef SmallVector<SharingMapTy, 4> StackTy;
```

Comment:
Can you split this into a separate commit? I'd like to have documented (in its commit message) why you did it. And then we'll send it upstream.